### ISSUE TITLE
Provide a simple way to force updates on SNAPSHOTs

### ISSUE THREAD

**Issue by rwinch** [eclipse/buildship#373]
It would be nice if there was a simple way within the UI to force downloads of SNAPSHOTs (both buildscript and project dependencies).

**Comment by oehme** [comment]
Agreed, this could be a second menu entry called `Refresh Gradle Project...` which opens a pop-up with advanced options.

**Comment by nickheniser** [comment]
Is there any workaround way to do this today?

**Comment by oehme** [comment]
No great one I'm afraid: Running a task like `assemble` and adding `--refresh-dependencies` in the "arguments" tab of the run configuration.

**Comment by clo-vis** [comment]
> Running a task like `assemble` and adding `--refresh-dependencies` in the "arguments" tab of the run configuration.
No effect.

> Is there any workaround way to do this today?
Restart Eclipse (File/Restart). 
Not many programs have "restart" as an option. If it has, then it must be important.

**Comment by mauromol** [comment]
I also miss this, because it seems like that a plain "Refresh Gradle Project" only actually updates SNAPSHOTs once in a day or such...
